### PR TITLE
Chef 18.5

### PR DESCRIPTION
**Pull request by tpowell-progress**
## Description
Bump to chef-foundation and tests to validate OpenSSL install.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] If `Gemfile.lock` has changed, I have used `--conservative` to do it and included the full output in the Description above.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: spec/integration/client/open_ssl_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe "openssl checks" do
+  let(:openssl_version_default) do
+    if windows? || aix?
+      "1.0.2zi"
+    elsif macos?
+      "1.1.1m"
+    else
+      "3.0.9"
```
Comment (Stromweld) [inline spec/integration/client/open_ssl_spec.rb:10]: shouldn't this be 3.2.0

Comment (tpowell-progress) [reply spec/integration/client/open_ssl_spec.rb:10]: @Stromweld no, 3.0.9 is the last FIPS-validated OpenSSL. The `openssl` gem's version is only loosely related.

Comment (Stromweld) [reply spec/integration/client/open_ssl_spec.rb:10]: I see

**Comment by tpowell-progress** [comment]
superseded by #14433

**Pull request closed without merging**